velox/expression/fuzzer/ExpressionFuzzerTest.cpp

```diff
@@ -76,6 +76,13 @@ int main(int argc, char** argv) {
       "regexp_like",
       "regexp_replace",
       "regexp_split",
+      // date_format and format_datetime throw VeloxRuntimeError when input
+      // timestamp is out of the supported range.
+      "date_format",
```

Comment:
Hmm, Since we have this same problem with casting , I wonder if we should add a feature in fuzzer to ignore system errors originating from certain functions ?